cibyl/yaml.py

```diff
@@ -0,0 +1,41 @@
+"""
```

Comment:
Maybe we can put files.py and yaml.py in cibyl/utils directory to distinguish them from core modules

Reply:
Will do.